requirements.txt

```diff
@@ -1 +1,7 @@
-fastapi>=0.62.0
\ No newline at end of file
+anyio==3.6.1; python_version >= "3.6" and python_full_version >= "3.6.2"
```

Comment:
Does this file need to be committed still with poetry?

Reply:
> The one caveat here is that the [tortoise-orm](https://github.com/tortoise/tortoise-orm) CRUDRouter is not available to those using python 3.6.

Oh ok, get it now — so if one uses python 3.6, then they simply cannot use tortoise-orm, is that correct?
If so, I'll update the code to accommodate the feature.

> Does this file need to be committed still with poetry?

It is not necessary, but allows people, who don't use poetry to simply install dependencies with `pip`, so I've kept it.

Reply:
> Oh ok, get it now — so if one uses python 3.6, then they simply cannot use tortoise-orm, is that correct?
If so, I'll update the code to accommodate the feature.

Yes this is correct!